index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1>Hello world we are software developers!</h1>
+    <h1 id="title"> Survey Form </h1>
```

Comment:
- [ ] I think it would be better to use only one h1 tag in your code for SEO reasons. The other h1 tag can be made an h2 tag or a p tag. You can leave the tag on line 11 and make changes to line 12.

Reply:
I have updated the comment to be more polite.